### PR TITLE
Enable stickers on all envs. Use contenthash to identify stickers

### LINKED ISSUE #8262: Use multihash for sticker image in message instead of uri
# Problem

Currently, it's possible to send any url in stickers

## Implementation

multihash should be sent and image should be loaded from ipfs or swarm

### PR DESCRIPTION
**Pull request by jeluard**
fixes #8262

### Summary

Changes how stickers are identified: from URL to content hash. 

Sticker market is enabled on all networks.
No user facing changes.

### Steps to test

- Open Status
- Test stickers can be installed, selected and sent

status: ready

### CODE REVIEW

**Review by vitvly — APPROVED** [review]
Probably makes sense to add tests for fns in `stickers.core` and `utils.contenthash` namespaces.

File: src/status_im/ui/screens/chat/stickers/views.cljs [hunk 1]
```diff
@@ -76,6 +78,9 @@
     (when pack-id
       (re-frame/dispatch [:stickers/select-pack pack-id]))))
 
+(defn- valid-sticker? [sticker]
```
Comment (vitvly) [inline src/status_im/ui/screens/chat/stickers/views.cljs]: Maybe this fn can be dropped in favour of `stickers.core.valid-stickers?`

File: src/status_im/utils/contenthash.cljs [hunk 2]
```diff
@@ -54,6 +54,12 @@
                            hex/decode
                            b58/encode))})))
 
+(defn url [hex]
+  (let [{:keys [namespace hash]} (decode (if (string/starts-with? hex "0x") hex (str "0x" hex)))]
```
Comment (flexsurfer) [inline src/status_im/utils/contenthash.cljs]: `normalize` function?

**Review by flexsurfer — APPROVED** [review]

File: src/status_im/ui/screens/chat/stickers/views.cljs (hunk 1 above)
Comment (jeluard) [reply src/status_im/ui/screens/chat/stickers/views.cljs]: :+1:

**Comment by asemiankevich** [comment]
@jeluard is it expected how sticker  details should look like? I mean this additional picture looks overkill

image

Also, i can't see a sticker on my device when sending from 1x1 chat. To reproduce:
1. install status on devices A and B
2. add devices A and B to contacts
3. install sticker pack Julien on ropsten on device A
4. send a sticker in 1x1 chat to device B
5. check 1x1 chat on device B

No incoming stickers shown

image

**Comment by jeluard** [comment]
>  I mean this additional picture looks overkill

@asemiankevich I don't think there can be too much of my face in status !

**Comment by jeluard** [comment]
@asemiankevich Sticker reception should be fixed now

**Comment by asemiankevich** [comment]
Now i can exchange stickers, and see them in chat preview. I can send stickers in 1x1 , public, group chat. 
Stickers are enabled across all the networks however we dont have any on Mainnet / Rinkeby .

image

**Pull request merged**